test/cmd/images-old-policy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc login -u system:admin
+  oadm policy reconcile-cluster-roles --additive-only=false --confirm
+  exit 0
+) &>/dev/null
+
+
+os::test::junit::declare_suite_start "cmd/images-old-policy"
+
+oc project -q
+project=$(oc project -q)
```

Comment:
> we enter this test in the correct project, but the oc login -u system:admin on line 15 lists available projects, and switches to "default" if the current project is not in the list. That races with the project authz cache seeing the new project

Huh?  That should run a sub-shell, which runs those commands, outputs both streams to dev/null and blocks until complete.  The `&` is tied to `&>`, so its a redirect, not a spawn.